### ISSUE TITLE
General Hyperparameter Policy (Batch Size, Shard Size, Learning Rate, etc.)

### ISSUE THREAD

**Issue by petermattson** [mlcommons/training_policies#66]
Discussed in Special Topics per this document
Approved general document
Clarify shard size as shuffling and packing 
Clarify where to request changes

**Comment by petermattson** [comment]
Will update rules per doc.

**Comment by hanlint** [comment]
@petermattson I don't see those updated yet. Is there another document I can reference? Thanks!
